dist/api_data.py

```diff
@@ -1162,6 +1162,16 @@ def __cmp__(self, other):
         type='list'),
 ]),
 
+'WT_SESSION.query_timestamp' : Method([
+    Config('get', 'read', r'''
+        specify which timestamp to query: \c read returns the timestamp the
+        transaction is reading at.  \c commit returns the most recently set
+        commit_timestamp. \c first_commit returns the first set
+        commit_timestamp. \c prepare returns the timestamp used in preparing a
+        transaction. See @ref transaction_timestamps''',
+        choices=['read', 'commit', 'first_commit', 'prepare']),
```

Comment:
Please alphabetize the choices and their order in the description above to also be alphabetized.